public/blog/first-ever-truffle-retreat.md

```diff
@@ -0,0 +1,123 @@
+# Our First Ever Truffle Retreat
+
+![Truffle](/blog/images/first-ever-truffle-retreat/medium-truffle-retreat.jpg)
+
+The Truffle team recently got together late September to define our roles, our mission statement, and our product roadmap - and had some fun in the process.  
+
+----------
+
+The Truffle team is fully remote. This has a lot of advantages in that it allows for a wide range of people available to work on your project; but it can also make it difficult to get to know your coworkers. 
+
+I started Truffle in early 2015 as the sole developer, and it stayed that way for nearly a year and half. John McDowell joined in late 2016, and by the time of the retreat we happily brought Nick D’Andrea and Josh Quintal on board. All interviews happened over Skype.
+
+I live in Yakima, Washington. John lives in Victoria BC. Nick lives in Philadelphia, and Josh lives in Detroit. That’s a lot of distance between us. 
+
+![Truffle employees in September](/blog/images/first-ever-truffle-retreat/truffle-map-01.jpg)
+
+When we brought Nick and Josh onto the team, it became obvious that we needed to meet up in person to solidify our roles, define responsibilities, and establish our own culture before growing the team even larger. 
+
+And we wanted to hang out in the woods, at the top of a mountain, and get to know each other. 
+
+## The Location
+
+![View from the house](/blog/images/first-ever-truffle-retreat/location.jpg)
+
+We had our retreat near Victoria, BC where John lives. Nick, Josh and I traveled all day through a series of planes, ferries and car rides to finally arrive at the house around 7 pm. The house sits atop a large hill in Braemer Heights overlooking all of Victoria, and on a good day has a beautiful view of Mt. Baker in the States. 
+
+The house was the definition of excess for a team of four, especially with only three of us sleeping there. It had five bedrooms and four bathrooms; a deck with a grill; three stories, the topmost containing only an office with 360-degree windows; and loads of wildlife. Unexpectedly, it was only ten minutes away from a Costco, which helped immensely. Even more unexpectedly, this mansion cost us less than three hotel rooms and a coworking space.  
+
+![Another view](/blog/images/first-ever-truffle-retreat/mountain.jpg)
+
+## The Work
+
+I had a few specific goals for our retreat: to define everyone’s roles and responsibilities within our team, to define our product roadmap for the next six months, and to create a mission statement for Truffle that our whole team could rally behind. 
```

Comment:
Feels like this list of priorities has been said already (L5, L17), unless your personal priorities differed somehow.